packages/table/preview/mutableTable.tsx

```diff
@@ -167,17 +177,20 @@ const DEFAULT_STATE: IMutableTableState = {
     enableCellEditing: false,
     enableCellSelection: true,
     enableCellTruncation: false,
+    enableCellWrap: false,
     enableColumnCustomHeaders: true,
     enableColumnNameEditing: false,
     enableColumnReordering: true,
     enableColumnResizing: true,
     enableColumnSelection: true,
     enableContextMenu: false,
     enableFullTableSelection: true,
+    enableLayoutBoundary: false,
     enableMultiSelection: true,
     enableRowReordering: false,
     enableRowResizing: false,
     enableRowSelection: true,
+    enableSlowLayout: false,
```

Comment:
@llorca FWIW the state-key names in this example have been a great testing ground for the ideas in #1625. Notice that we have both `enable...` and `show...` happening here. We really just need the former I think.

(No action required; just FYI)